src/sentry/api/endpoints/assistant.py

```diff
@@ -8,10 +8,11 @@
 from rest_framework.response import Response
 
 from sentry.api.base import Endpoint
-from sentry.assistant.guides import GUIDES
 from sentry.models import AssistantActivity
+from sentry.assistant import manager
 
 
+GUIDES = manager.all()
 VALID_GUIDE_IDS = frozenset(v['id'] for v in GUIDES.values())
```

Comment:
This is problematic now because manager may not have all the guides when this line is executed. Imagine if this module is loaded before getsentry can put its guides in the manager. So just call `get_valid_ids` in the endpoint handler and remove this constant.